src/lib/store/remote/helpers.js

```diff
@@ -174,12 +174,13 @@ exports.mapDocsFromFindAll = function(state, response) {
 // not return a longpoll URL, as we want it to finish right away, even if there
 // are no changes on remote.
 //
-exports.pullUrl = function(state) {
+exports.pullUrl = function(state, options) {
   var since = state.remote.getSinceNr();
+  var forceGzip = options && !!options.forceGzip
```

Comment:
Again `gzip=force`. My thinking was so that we can do `gzip=off` (force no gzip regardless of headers) and `gzip=headers` (default) later without having to introduce more parameters.